cmd/checkcommits/checkcommits.go

```diff
@@ -582,7 +582,7 @@ func NewCommitConfig(needFixes, needSignOffs bool, fixesPrefix, signoffPrefix, i
 	}
 
 	if config.NeedFixes {
-		config.FixesPattern = regexp.MustCompile(fmt.Sprintf(`(?i:%s\s*:?\s*#\d+)`, config.FixesString))
+		config.FixesPattern = regexp.MustCompile(fmt.Sprintf(`(?i:%s\s*:?\s*(#\d+|github\.com/kata-containers/[a-z-.]*#\d+))`, config.FixesString))
```

Comment:
I'd be tempted to define the new part of this regex as a const (like `versionSuffix`).